lib/libraries/cms/controller/form.php

```diff
@@ -298,7 +298,7 @@ public function cancel($key = null)
 		$app = JFactory::getApplication();
 		$model = $this->getModel();
 		$table = $model->getTable();
-		$checkin = property_exists($table, 'checked_out');
```

Comment:
@ercanozkaya You are right, there was a typo in the logic. I have fixed this and everything should now be working. Can you review ?